### DISCUSSION TITLE
Showing error while clicking 'Submit Label' in QuPath

### DISCUSSION THREAD

**Discussion by Suchi97** [Project-MONAI/MONAILabel#906] [Q&A] [answered]
Hi Team, 

We are trying to annotate the pathology labels. Here, we used QuPath for the annotation.
So we used the same images in local machine and for MONAI Label server (MONAI Label server is running in DGX machine).
We annotated the image as given below:
![image](https://user-images.githubusercontent.com/25792114/182361912-efd69465-0514-46e2-a47d-03a41e8fec0d.png)
And then clicked `Submit Label`. 
Then it showing the below error:
![image](https://user-images.githubusercontent.com/25792114/182362121-00755d13-e7af-47e1-9066-fa22d80a7d8b.png)

Can you please help me to solve this issue.
Any help is appreciated

Thanks in advance

**Comment by Suchi97** [comment, marked as answer]
Hi Team, 

The issue is solved by removing the extra forward slash 

In the preference section, I added `http://hostname:8000/`
So removing the forward slash solves the above issue.

**Reply by diazandr3s** [reply]
Thanks for letting us know :)
